target/linux/layerscape/patches-4.4/0001-arch-Introduce-post-init-read-only-memory.patch

```diff
@@ -0,0 +1,114 @@
+From 7ece66e296972744ec471d521b8ca07990e2494f Mon Sep 17 00:00:00 2001
```

Comment:
@hauke , 
Almost all these patches are for layerscape IP modules. We are also working on kernel 4.9 now. Even we migrate to kernel 4.9, I'm afraid the number of patches would not shrunk so much. 

The large number patches mainly comes from dts (155 patches), mc-bus (139 patches), sec (138 patches), qspi (80 patches). What I can do is to merge each IP module patches into one. We can provide only one patch for each module/component. Then we will only have dozens of patches finally.

It's also easy for maintaining, becasue we have each module as a feature branch when backport patches.

Do you think is it ok? Let me generate a new version for your reveiwing.
Thank you very much.